msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md

```diff
@@ -0,0 +1,236 @@
+
+```json
+{
+    "meetingRole":"<meeting role>",
+    "conversation":
+        {
+            "id": "<conversation id>"
+        }
+}
+```
+
+**meetingRole** can be *organizer*, *presenter*, or *attendee*.
+
+**Example 2**
+
+```json
+{
+    "meetingRole": "Attendee",
+    "conversation": {
+        "isGroup": true,
+        "id": "sample" }
+}
+```
+
+#### Response Codes
+
+**200**: participant information successfully retrieved  
+**401**: invalid token  
+**403**: the app is not allowed to get participant information. There can be many reasons: app disabled by tenant admin, blocked during live site mitigation, etc.    
+**404**: the meeting doesn't exist or participant can’t be found    
+
+<!-- markdownlint-disable MD024 -->
+### NotificationSignal API
+
+#### Request
+
+```http
+POST /v3/conversations/{conversationId}/activities
+```
+
+#### Query parameters
+
+**conversationId**: The conversation identifier. Required
+
+#### Request Payload
+
+**Example 1**
+
+```json
+{
+"channelData": {
+    "notification": {
+        "alertInMeeting": true,
+
+        "externalResourceUrl: "https://teams.microsoft.com/l/bubble/APP_ID?url=<TaskInfo.url>&height=<TaskInfo.height>      &width=<TaskInfo.width> &title=<TaskInfo.title> ”
+                      }
+                 }
+            }
+```
+
+**Example 2**
+
+```nodejs
+const replyActivity = MessageFactory.text('Hi'); // this could be an adaptive card instead
+        replyActivity.channelData = {
+            notification: {
+                alertInMeeting: true,
+                externalResourceUrl: 'https://teams.microsoft.com/l/bubble/APP_ID?url=<TaskInfo.url> &height=<TaskInfo.height>      &width=<TaskInfo.width> &title=<TaskInfo.title>’
+            }
+        };
+        await context.sendActivity(replyActivity);
+```
+
+**Example 3**
+
+```csharp
+var replyActivity = MessageFactory.text('Hi'); // this could be an adaptive card insteadreplyActivity.ChannelData = new TeamsChannelData{
+    Notification: new NotificationInfo(alertInMeeting: true, externalResourceUrl: 'https://teams.microsoft.com/l/bubble/APP_ID?url=<TaskInfo.url> &height=<TaskInfo.height>      &width=<TaskInfo.width> &title=<TaskInfo.title>’
+};
+await context.sendActivity(replyActivity);
+```
+
+#### Response Codes
+
+**201**: activity with signal is successfully sent  
+**401**: invalid token  
+**403**: the app is not allowed to send the signal. In this case, the payload should contain more detail error message. There can be many reasons: app disabled by tenant admin, blocked during live site mitigation, etc.  
+**404**: meeting chat doesn't exist  
+
+#### GetUserContext
+
+Please refer to our [Get context for your Teams tab](../tabs/how-to/access-teams-context.md#getting-context-by-using-the-microsoft-teams-javascript-library) documentation for guidance on identifying and  retrieving contextual information for your tab content. As part of meetings extensibility, new values have been added for the request payload:
```

Comment:
> new values have been added for the request payload

I'm not sure how to interpret the table below. The only new value that we added is "meetingId". 
tid and userObjectId are existing properties.

Also, the 3 entries aren't consistent, making it hard to decipher what's the description and what the actual property name is
**meetingId**
**tenantId - tid**
**participantId** - userObjectId